### PR TITLE
Chip Styles

### PR DESCRIPTION
**Pull request by jordanell**
## Description

1. Updated chip grey colors available
2. Updated chip disabled styles
3. Updated chip focus styles
4. Updated chip clickable styles
5. Updated tests / docs for chip

## Where to Start

The chip component. It's pretty heavily refactored to might be best to just examine the whole file rather than the diff.

### CODE REVIEW

File: packages/riipen-ui/src/components/Chip.jsx [hunk 1]
```diff
@@ -1,330 +1,399 @@
...
+    onBlurVisible();
+  };
 
-    /**
-     * If `true`, the chip should be displayed in a disabled state.
-     */
-    disabled: PropTypes.bool,
+  const handleClick = () => {
+    if (onClick) onClick();
+  };
```
Comment (ksmort) [inline packages/riipen-ui/src/components/Chip.jsx:48]: Can onClick just get passed into the component?

File: packages/riipen-ui/src/components/Chip.jsx [hunk 2]
```diff
@@ -1,330 +1,399 @@
...
+          background-color: ${theme.palette.grey[300]};
+          border-radius: 50px;
+          border: 1px solid;
+          border-color: transparent;
+          box-sizing: border-box;
+          color: ${theme.palette.text.secondary};
+          cursor: default;
+          display: inline-flex;
+          font-family: ${theme.typography.fontFamily};
+          font-size: 13px;
```
Comment (ksmort) [inline packages/riipen-ui/src/components/Chip.jsx:78]: Is it possible to use one of our preset font sizes here?

File: packages/riipen-ui/src/components/Chip.jsx [hunk 3]
```diff
@@ -1,330 +1,399 @@
...
+          white-space: nowrap;
+        }
+        .root.xsmall {
+          padding: 1px 5px;
+        }
+        .root.small {
+          padding: 5px 10px;
+        }
+        .root.medium {
+          padding: 10px 15px;
```
Comment (ksmort) [inline packages/riipen-ui/src/components/Chip.jsx:91]: Can use theme.spacing for some of these

File: packages/riipen-ui/src/components/Chip.jsx [hunk 4]
```diff
@@ -1,330 +1,399 @@
...
+        .root.clickable:hover,
+        .root.focusVisible,
+        .root.hover:hover {
+          background-color: transparent;
+          border: 1px solid ${theme.palette.text.primary};
+        }
+        .icon {
+          align-items: center;
+          display: flex;
+          padding-right: 10px;
```
Comment (ksmort) [inline packages/riipen-ui/src/components/Chip.jsx:109]: ditto theme spacing

File: packages/riipen-ui/src/components/Chip.jsx [hunk 5]
```diff
@@ -1,330 +1,399 @@
...
+          align-items: center;
+          display: flex;
+          white-space: nowrap;
+        }
 
-    /**
-     * Action to perform when clicked.
-     */
-    onClick: PropTypes.func,
+        .primary {
```
Comment (ksmort) [inline packages/riipen-ui/src/components/Chip.jsx:119]: What's our rule on whitespace between css in the ui kit?  Some of these have newlines between them, some don't

**Review by ksmort — CHANGES_REQUESTED** [review]

File: packages/riipen-ui/src/components/Chip.jsx (hunk 5 above)
Comment (jordanell) [reply packages/riipen-ui/src/components/Chip.jsx:119]: We actually don't really have a rule. I just try to group them to what makes sense given the context.

**Review by ksmort — APPROVED** [review]
🙌

**Pull request merged**